### DISCUSSION TITLE
Cannot log in on new phone with 2fa

### DISCUSSION THREAD

**Discussion by justingolden21** [community/community#144020] [Mobile]
I've been at this for several hours now. It wants an auth code and there's no way to access it. I've tried on my old phone and on the website. I know my username and password. I tried creating a passkey. This is by far the worst 2fa experience of my lifetime. Does anyone know how I can sign in to the app on my new phone? (Both phones are pixel 8)

**Comment by marlonwq** [comment]
Hello, you could try disabling authentication, [explanatory article](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account), and then enabling it on the new device after logging in

**Reply by justingolden21** [reply]
Thank you for the quick reply. I ended up getting it after trying many many many times, finally an option to have a number display and confirm in app appeared (this option was never available in all previous attempts and I did nothing differently to make it appear) and it worked.

**Reply by marlonwq** [reply]
Amazing, I'm glad you made it
